content/api/power-board.md

```diff
@@ -19,4 +19,29 @@ r.power_board.power_off()
 `power_on` is called when you setup your robot, so this doesn't need to be called manually. The ports will come on automatically as soon as your robot is ready, before the start button is pressed.
 {{% /notice %}}
 
+## Buzzing :bee:
+
+The power board has a piezo buzzer which can buzz.
+
+The `buzz` function accepts multiple parameters, depending on what you want to play. The first argument is the duration of the beep, in seconds. The later arguments are either the note you want to play, or the buzzers frequency (in Hz).
```

Comment:
s/buzzers/buzzer's/, though I actually think it would be clearer phrased as "frequency of the buzz you want" or similar (something about it being a future action makes the use of "of" in the latter feel nicer to me than the possessive in the former)